docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -129,9 +129,9 @@ More details can be found in the corresponding How to Guides, which have been li
   ]}>
 <TabItem value="gcs">
 
-To connect to your data in GCS, first instantiate your project's DataContext by importing the necessary packages and modules. 
+To connect to your data in GCS, first instantiate your project's DataContext by importing the necessary packages and modules.
 
-```python file=../../tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs_yaml_configs.py#L4-L5
+```python file=../../tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs_yaml_configs.py#L4-L6
```

Comment:
This is the big thing to focus on - the line number changes here are used to proper line number referencing.

If we have a line number range, the immediate preceeding and following lines should have `<snippet>` and `</snippet>` in them, respectively.